test/functional/lua/interrupts_spec.lua

```diff
@@ -187,6 +192,14 @@ describe(':luado command', function()
     feed(':cq<CR>')
     screen_check_exit(screen)
   end)
+  it('may be interrupted with <C-c> without additional :terminal layer',
+  function()
+    feed('iabc<CR>def<CR>ghi<Esc>:%luado while true do i = nil end<CR>')
+    sleep(100)
+    feed('<C-c><C-c><C-c>')
```

Comment:
any idea why 3 tries are needed here, but not in the others? Comment in the test would be helpful.